src/App.js

```diff
@@ -36,20 +37,60 @@ class App extends React.Component {
                dataByStatuses[status].push(card);
             });
 
-            console.log(stateObj);
+            //console.log(stateObj);
 
             this.setState(stateObj);
         } catch (error) {
             alert("Ошибка HTTP: " + error);
         }          
     }
 
+    handleCreateCard = async(description) => {        
+        const urlCreateCard = this.props.urlCreateCard;
+        const {statuses, dataByStatuses} = this.state;
+        let body = {
+            description
+        }
+
+        try {
+            let response = await fetch (urlCreateCard, {
+                method: 'POST',
+                headers: {
+                    'Content-Type': 'application/json;charset=utf-8'
+                },
+                body: JSON.stringify(body)            
+            });
+
+            let result = await response.json();                      
+
+            let updateDataByStatuses = this.state.dataByStatuses;
+            updateDataByStatuses[result.status].push(result);
```

Comment:
`push` will modify state array, please don't do it!
to avoid this you need to create another variable with spreaded array:
let updateDataByStatuses = {
  ...dataByStatuses,
  [result.status]: [...dataByStatuses[result.status], result]
}